src/updater.cpp

```diff
@@ -149,7 +150,7 @@ void Updater::setupOnWindows()
 
     // finally, create the dblsqd objects. Constructing the UpdateDialog triggers the update check
     updateDialog = new dblsqd::UpdateDialog(feed, updateAutomatically() ? dblsqd::UpdateDialog::Manual : dblsqd::UpdateDialog::OnLastWindowClosed, nullptr, settings);
-    installOrRestartButton = new QPushButton(tr("Update"));
+    installOrRestartButton->setText(tr("Update"));
```

Comment:
Isn't this new line redundant as the text is already given as an argument to the constructor in the `Updater` class constructor?

Reply:
The text can change - there are other `setText()` calls in the code - so I'd just like to ensure here that it is what it ought to be.